src/components/Home/CardItem.js

```diff
@@ -24,9 +24,14 @@ const CardItem = (props) => {
         <div class="card-body">
           <h5 class="card-title">{props.title}</h5>
           <p class="card-text">{props.text}</p>
-          <Link to={props.path} class="btn btn-primary">
+          <ScrollLink
+            to={props.path}  
+            smooth={true}
+            duration={500}
+            className="btn btn-primary"
+          >
             Play
-          </Link>
+          </ScrollLink>
```

Comment:
This is supposed to take the user to the page of the particular game/activity so scroll link is not required here